amads/time/meter/attractor_tempos.py

```diff
@@ -0,0 +1,200 @@
+"""
+What does it mean for music to be "fast" or "slow"?
+Certainly BPM is not enough.
+The "Attractor tempos" theory (Gotham 2015, [1]) proposes
+ a definition of "fast"/"slow" relative to neutral, central, moderate tempos
+ and a definition of those moderate ("Attractor") tempos that accounts for the role of the metrical structure.
+In short, it provides a model for optimizing the salience of metrical structures.
+
+[1] Gotham, M. (2015). Attractor tempos for metrical structures. Journal of Mathematics and Music, 9(1), 23–44.
+https://doi.org/10.1080/17459737.2014.980343
+"""
+
+__author__ = "Mark Gotham"
+
+from typing import Optional, Union
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+# ------------------------------------------------------------------------------
+
+
+class MetricalSalience:
+    """
+    Organise array representations of metrical structre and derived salience values.
```

Comment:
typo

Reply:
Ok. Debatable, but changed.

Done.

Reply:
I meant that you had misspelled "structre"!

Reply:
Oh in that case, not debatable!

:)

I wonder why the (usually overzealous) spell checker missed that.